Gemfile

```diff
@@ -4,5 +4,8 @@ group :test do
   gem 'rake'
   gem 'rspec'
   gem 'beaker', '~>1.17.0'
+  if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
```

Comment:
I didn't go with string equality because I thought it would be vaguely possible that in the future for some reason our internal rubygem mirror requires `https`, the regex matches both https://rubygems.delivery.puppetlabs.net and http://rubygems.delivery.puppetlabs.net